third_party/uvwasi/CMakeLists.txt

```diff
@@ -42,6 +42,11 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
   list(APPEND uvwasi_defines _GNU_SOURCE _POSIX_C_SOURCE=200112)
 endif()
 
+if (WALRUS_MODE STREQUAL "debug")
+    list(APPEND uvwasi_defines UVWASI_DEBUG_LOG)
+    add_compile_options( -O0 -g )
```

Comment:
this option will affect the entire project build.
Is there another way to change the `uvwasi` build only?